configuration/scripts/tests/timeseries.py

```diff
@@ -0,0 +1,275 @@
+    logger.info('Saving file to {}'.format(outfile))
+    plt.savefig(outfile,dpi=300,bbox_inches='tight')
+
+def main():
+    import argparse
+    parser = argparse.ArgumentParser(description="To generate timeseries plots, this script \
+                                     can be passed a directory containing a logs/ subdirectory, \
+                                     or it can be run in the directory with the log files, \
+                                     without being passed a directory.  It will pull the \
+                                     diagnostic data from the most recently modified log file.\
+                                     \
+                                     If no flags are passed selecting the variables to plot, \
+                                     then plots will be created for all available variables.")
+    parser.add_argument('log_dir', nargs='?', default=os.getcwd(), \
+                        help="Path to diagnostic output log file directory.")
+    parser.add_argument('--bdir',dest='base_dir', help='Path to the directory that contains \
+                         the log file for a baseline dataset, if desired.')
+    parser.add_argument('-v', '--verbose', dest='verbose', help='Print debug output?', \
+                        action='store_true')
+    parser.add_argument('--area', dest='area', help='Create a plot for total ice area?', \
+                        action='store_true')
+    parser.add_argument('--extent', dest='extent', help='Create a plot for total ice extent?', \
+                        action='store_true')
+    parser.add_argument('--volume', dest='ice_volume', help='Create a plot for total ice volume?', \
+                        action='store_true')
+    parser.add_argument('--snw_vol', dest='snow_volume', help='Create a plot for total snow \
+                        volume?', action='store_true')
+    parser.add_argument('--speed', dest='speed', help='Create a plot for rms ice speed?', \
+                        action='store_true')
+    parser.add_argument('--grid',dest='grid', help='Add grid lines to the figures?', \
+                        action='store_true')
+
+    # Set the defaults for the command line options
+    parser.set_defaults(verbose=False)
+    parser.set_defaults(area=False)
+    parser.set_defaults(extent=False)
+    parser.set_defaults(ice_volume=False)
+    parser.set_defaults(snow_volume=False)
+    parser.set_defaults(speed=False)
+    parser.set_defaults(grid=False)
+
+    args = parser.parse_args()
+
+    # If no fields are passed, plot all fields
+    if not ( args.area or args.extent or args.ice_volume or args.snow_volume or args.speed ):
+        args.area = True
+        args.extent = True
+        args.ice_volume = True
+        args.snow_volume = True
+        args.speed = True
+
+    # Build the fieldlist based on which fields are passed
+    fieldlist = []
+    if args.area:
+        fieldlist.append('total ice area  (km^2)')
```

Comment:
It would be nice if the graph title and the ylabel would format the `^2`, `^3` as exponents.